python_modules/dagster-graphql/dagster_graphql/schema/pipelines/pipeline.py

```diff
@@ -75,6 +75,14 @@ def resolve_partition(self, _graphene_info):
         return self._event.dagster_event.step_materialization_data.materialization.partition
 
 
+class GrapheneMaterializationCountByPartition(graphene.ObjectType):
```

Comment:
Maybe this can just be GrapheneMaterializationCount with an optional partition?  Then we could reuse it if we wanted to do non-partitioned materialization counts.

Reply:
sounds good!